objects/screen.h

```diff
@@ -46,6 +51,31 @@ struct a_screen
 };
 ARRAY_FUNCS(screen_t *, screen, DO_NOTHING)
 
+/**
+ * This structure tracks the area where there is physical screens.
```

Comment:
If an area can cross physical screen boundaries, then `is` --> `are`.
If an area cannot cross physical screen boundaries, then `is a physical screen`.